fake_service/schema/utils.py

```diff
@@ -7,21 +7,21 @@
 fake = Faker()
 
 
-def make_generetions(data: List, count: str) -> List:
-    list2 = []
-    for i in range(int(count)):
-        dict1 = {}
+def generate_fake_data(data: List[Dict[str, Any]], row_nums: str) -> List:
+    generated_data = []
+    for i in range(int(row_nums)):
+        column = {}
         for item in data:
             name = item["name"]
             if item["type"] == "integer":
-                dict1[name] = random.randint(
+                column[name] = random.randint(
                     int(item["value_from"]), int(item["value_to"])
                 )
             if item["type"] != "integer":
                 column_type = getattr(fake, item["type"])
-                dict1[name] = column_type()
-        list2.append(dict1)
-    return list2
+                column[name] = column_type()
+        generated_data.append(column)
+    return generated_data
```

Comment:
this function looks like service to me, move to services. Alse it would be great to encapsulate it along with fake(move to init)
FE:
```
class FakeDataGenerator:
    def __init__(self):
        faker = Faker()
        
    def generate_fake_data(self, generate_fake_data(data: List[Dict[str, Any]], row_nums: str) -> List:):
        ...
```